src/main/java/build/buildfarm/common/redis/RedisNodeHashes.java

```diff
@@ -78,7 +79,7 @@ public static List<String> getEvenlyDistributedHashesWithPrefix(
     if (jedis instanceof JedisCluster cluster) {
       Iterable<List<List<Long>>> nodeSlotRanges = getNodeSlotRanges(cluster);
       try {
-        ImmutableList.Builder hashTags = ImmutableList.builder();
+        ImmutableList.Builder<String> hashTags = ImmutableList.builder();
```

Comment:
oof on both of these. Would be great if we had static analysis that told us when we misuse generics like this.